GNUmakefile

```diff
@@ -154,29 +154,29 @@ lint-deps: ## Install linter dependencies
 .PHONY: check
 check: ## Lint the source code
 	@echo "==> Linting source code..."
-	@gometalinter \
-		--deadline 10m \
-		--vendor \
-		--exclude='.*\.generated\.go' \
-		--exclude='.*bindata_assetfs\.go' \
-		--skip="ui/" \
-		--sort="path" \
-		--aggregate \
-		--enable-gc \
-		--disable-all \
-		--enable goimports \
-		--enable misspell \
-		--enable vet \
-		--enable deadcode \
-		--enable varcheck \
-		--enable ineffassign \
-		--enable structcheck \
-		--enable unconvert \
-		--enable gas \
-		--enable gofmt \
-		./...
+	# @gometalinter \
+	# 	--deadline 10m \
+	# 	--vendor \
+	# 	--exclude='.*\.generated\.go' \
+	# 	--exclude='.*bindata_assetfs\.go' \
+	# 	--skip="ui/" \
+	# 	--sort="path" \
+	# 	--aggregate \
+	# 	--enable-gc \
+	# 	--disable-all \
+	# 	--enable goimports \
+	# 	--enable misspell \
+	# 	--enable vet \
+	# 	--enable deadcode \
+	# 	--enable varcheck \
+	# 	--enable ineffassign \
+	# 	--enable structcheck \
+	# 	--enable unconvert \
+	# 	--enable gas \
+	# 	--enable gofmt \
+	# 	./...
 	@echo "==> Spell checking website..."
-	@misspell -error -source=text website/source/
+	# @misspell -error -source=text website/source/
```

Comment:
Ha, I don't think you meant to commit this. :)

Reply:
D'oh! Fixed.